ups_install/installTableFile.sh

```diff
@@ -62,7 +62,8 @@ Common:
 
     exeActionRequired(GetFQDir)
     envSet (OFFLINE_LIB, \${OFFLINE_DIR}/\${OFFLINE_FQ}/lib )
-    pathAppend( CET_PLUGIN_PATH, \${OFFLINE_LIB} )
+    pathAppend( CET_PLUGIN_PATH, \${OFFLINE_LIB} )    
+    pathPrepend(PATH, "\${OFFLINE_DIR}/\${OFFLINE_FQ}/bin")
```

Comment:
I just setup the product and tried

ls ${OFFLINE_DIR}/${OFFLINE_FQ_DIR}/bin

It looks like Offline_FQ is not defined correctly - it's missing the "trigger" piece.

Reply:
This is ok it was a mistake on my part - see the conversation.  Resolving this now.